src/main/resources/add-ons-to-index.json

```diff
@@ -1,5 +1,26 @@
 {
   "toIndex": [
+    {
+      "uid": "org.openmrs.owa.orderentryui",
+      "type": "OWA",
+      "name": "Order Entry UI OWA",
+      "maintainers": [
+        {
+          "name": "Andela apprentices",
+          "url": "https://andela.com/"
+        },
+        {
+          "name": "OpenMRS",
+          "url": "https://github.com/openmrs"
+        }
+      ],
+      "backend": "org.openmrs.addonindex.backend.Bintray",
+      "bintrayPackageDetails": {
+        "owner": "openmrs",
```

Comment:
According to Bintray, you i.e. larrystone should be the owner. Hence, kindly change this from openmrs to larrystone

Reply:
Thanks for this observation. 
I am currently working on been added to the openmrs bintray account. When it gets approved, I will be creating the package directly on the openmrs owa repo, removing the existing link to mine

Reply:
I have been granted the permission and I have created the package on the openmrs organisation's owa repo. @reubenvarghese1

Reply:
Great thanks, I am merging this PR :)